services/models/workload.go

```diff
@@ -23,6 +23,19 @@ type WorkloadOverview struct {
 	// required: true
 	// example: deployment
 	Type string `json:"type"`
+
+	// Has IstioSidecar
```

Comment:
If this is for swagger, it should have a little bit more workding to not confuse users.
It can say something like "Define if Pods related to this Workload has an IstioSidecar deployed".

Just to avoid the confusion that a Sidecar is not par of a Workload/Deployment per se.